### ISSUE TITLE
[Game Support]: Fifa mobile

### ISSUE THREAD

**Issue by Atur14** [PlayCover/PlayCover#359]
### App Store link

https://apps.apple.com/us/app/fifa-soccer/id1094930513

### Current issues

- [ ] Game fails to install
- [ ] Game fails to open
- [ ] Game crashes randomly during gameplay
- [ ] Game does not work with keymapping
- [X] Game experiences graphical issues or broken UI
- [ ] Game fails to connect to servers or complete download

### More details

When I open the game, no text is shown. The buttons i click on, also do not work, and as a result i am stuck on this part of the game.
![image](https://user-images.githubusercontent.com/113869948/190939323-47931827-cbe2-43e6-9ce2-825492c5c0b8.png)

### Crash log

```shell
It does not crash, but it just simply does not work
```

### Issue Language

- [X] Yes my issue is written in English

**Comment by Atur14** [comment]
Describe the bug
When I open the game, no text is shown. The buttons i click on, also do not work, and as a result i am stuck on this part of the game.

Steps to reproduce
Just use the app

Expected behaviour
Please make text appear

Crash log
No response

What version of PlayCover are you using?
1.1.1

What version of macOS are you using? 12.6

**Comment by IsaacMarovitz** [comment]
@Atur14 Have you tried disabling SIP

**Comment by Atur14** [comment]
yes but it didnt work

**Comment by IsaacMarovitz** [comment]
@Atur14 Try uninstalling, and installing again with the latest nightly.

**Comment by EvilAirborn** [comment]
what is the update? I am still facing the issue

**Comment by TheMoonThatRises** [comment]
Have you tried using the latest nightly build? This issue is several months old and many changes have been made since.

**Comment by EvilAirborn** [comment]
Yes i tried the nighty builds! Just few moments ago, same issue. Followed all the guidelines tpp
